### ISSUE TITLE
Last layer activation

### ISSUE THREAD

**Issue by rashemihmih** [raghakot/keras-resnet#49]
Maybe you should use sigmoid activation instead of softmax for the last layer if num_outputs == 1?

**Comment by raghakot** [comment]
True..I think i should just leave out the dense layer as flexible since it can be used for regression as well.
